stock_by_warehouse_sale/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "Stock by Warehouse on Sales",
+    "version": "12.0.1.0.0",
```

Comment:
Also, in here a big bump in the version is needed @hugho-ad

Reply:
for sure